make_realse.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
```

Comment:
Great work here. I propose we move this to a `scripts/` directory, the users should not use it, instead they should download realeases from our repo's Releases section where we can build them for each OS via a GitHub Action using your bash script.

This is for beyond v2.0. The deployment model should still be git clone/pull for now. Once we fine tune the release system we can add download links to the README.

Reply:
Typo in the name `make_realse` instead of `release`.

Reply:
ok, i needed a release package for my needs asap, thats why i made the script so early :)
no pressure on you, it's a fork